ovs/lib/tests/test_mdsservice.py

```diff
@@ -227,7 +227,7 @@ def _create_vdisks_for_mds_service(self, amount, start_id, mds_service=None, vpo
         """
         vdisks = {}
         storagedriver_id = None
-        if mds_service is not None:
+        if mds_service is not None and mds_service.service.storagerouter_guid is not None:
```

Comment:
It should use the dynamic property, and is this even required? It looks like a silent ignore here, as the storagerouter is mandatory for an MDS service.